src/tools/error_index_generator/book_config.toml

```diff
@@ -0,0 +1,17 @@
+[book]
+title = "Error codes index"
+description = "Book listing all Rust error codes"
+src = ""
+
+[output.html]
+git-repository-url = "https://github.com/rust-lang/rust/"
+
+[output.html.search]
+enable = true
+limit-results = 20
+use-boolean-and = true
+boost-title = 2
+boost-hierarchy = 2
+boost-paragraph = 1
+expand = true
+heading-split-level = 2
```

Comment:
To be noted: the size issue was a problem because it was an HTML page which made the rendering very slow and problematic on mobile devices. In this case, it's "better" considering it's not loaded by default and is not part of the DOM.